ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/claimantresponse/ResponseAcceptationAssert.java

```diff
@@ -1,12 +1,13 @@
-package uk.gov.hmcts.cmc.ccd.deprecated.assertion.claimantresponse;
+package uk.gov.hmcts.cmc.ccd.assertion.claimantresponse;
 
 import org.assertj.core.api.AbstractAssert;
-import uk.gov.hmcts.cmc.ccd.deprecated.domain.claimantresponse.CCDResponseAcceptation;
+import uk.gov.hmcts.cmc.ccd.domain.claimantresponse.CCDResponseAcceptation;
 import uk.gov.hmcts.cmc.domain.models.claimantresponse.ResponseAcceptation;
 
 import java.util.Objects;
 
-import static uk.gov.hmcts.cmc.ccd.deprecated.assertion.Assertions.assertThat;
+import static uk.gov.hmcts.cmc.ccd.assertion.Assertions.assertThat;
+
 
 public class ResponseAcceptationAssert extends AbstractAssert<ResponseAcceptationAssert, ResponseAcceptation> {
```

Comment:
that assertion can't be added here because the submittedOn field is not available in the cmc ResponseAcceptation ....I have since modified the test case and added assertion there ..please have a look at ClaimantResponseMapperTest